src/Sylius/Bundle/ShopBundle/Resources/config/services/twig/component.xml

```diff
@@ -89,5 +89,57 @@
                 template="@SyliusShop/product/common/original_price.html.twig"
             />
         </service>
+
+        <service
+            id="sylius_shop.twig.component.product_summary"
+            class="Sylius\Bundle\ShopBundle\Twig\Component\Product\ProductSummaryComponent"
```

Comment:
```suggestion
            class="Sylius\Bundle\ShopBundle\Twig\Component\Product\SummaryComponent"
```
As it is in the `Product` directory, there is no need for additional prefix in class name